src/app/components/ForecastCard/ForecastCard.module.css

```diff
@@ -0,0 +1,26 @@
+.forecastCard {
+  font-size: 2em;
+  color: var(--clr-text-secondary);
+  border-radius: 10px;
+  background: #9dc3e230;
+  backdrop-filter: blur(5px) saturate(80%);
```

Comment:
FYI: This feature is not available to all major browser right now. Firefox for example need a special setting to use it.
Read more [here](https://caniuse.com/css-backdrop-filter).